docker/Dockerfile

```diff
@@ -54,85 +100,86 @@ RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/inst
 RUN poetry config virtualenvs.create false && \
     poetry config installer.parallel false
 
-# Keep the project source code from NAUTOBOT_ROOT
-COPY pyproject.toml poetry.lock /source/
+COPY pyproject.toml poetry.lock README.md /source/
+
+# The example_plugin is only a dev dependency, but Poetry fails to install non-dev dependencies if its source is missing
 COPY examples /source/examples
+
 WORKDIR /source
 
-# -------------------------------------------------------------------------------------
-# Install Nautobot requirements
-# -------------------------------------------------------------------------------------
-# During image build process, only Nautobot's requirements are installed.
-#
-# Local source code is mounted as volume into container. This approach does not require
-# to rebuild a container when source code change occurs.
-# -------------------------------------------------------------------------------------
-RUN poetry install --no-root --no-ansi --extras all
+# Install (non-development) Python dependencies of Nautobot
+# pyuwsgi wheel doesn't support ssl so we build it from source
+# https://github.com/nautobot/nautobot/issues/193
+RUN pip install --no-cache-dir --no-binary=pyuwsgi pyuwsgi==${PYUWSGI_VER} && \
```

Comment:
Managing outside of poetry to account for an arg of uwisgi version I assume?

Reply:
`pip` is used rather than `poetry` because poetry doesn't currently have a way to force non-binary installation of a dependency. 

The `PYUWSGI_VER` arg is used so as to ensure that we install the expected version of pyuwsgi matching what's specified in `pyproject.toml` - in the "before" version of Dockerfile there was no constraint on the pyuwsgi version in the Docker build, it'd just install whatever happened to be latest.